### ISSUE TITLE
Problem: Reingest metadata only can fail for certain processing configurations

### ISSUE THREAD

**Issue by jorikvankemenade** [archivematica/Issues#959]
**Please describe the problem you'd like to be solved**
When you update the default processing to process transfers fully automatically, I.e. setting "Reminder: add metadata if desired:" to `continue`, the metadata reingest only will not work. The only point at which the ingestion will stop is "Approve reingest", but if you update the metadata at that point the metadata will not be reingested. 

**Describe the solution you'd like to see implemented**
I think the simplest, most UX friendly, solution is to add a processing.xml that is just for the metadata reingest. This way changing the default, will not break core functionality.

**Describe alternatives you've considered**
Make sure that in the metadata reingest workflow you always pause for adding metadata.
From a UX perspective, this is also very helpful, but I don't know how invasive this change is. 

**Additional context**
Archivematica 1.9 and Storage Service 0.15. 

---

**For Artefactual use:**

Before you close this issue, you must check off the following:

- [ ] All pull requests related to this issue are properly linked
- [ ] All pull requests related to this issue have been merged
- [ ] A testing plan for this issue has been implemented and passed (testing plan information should be included in the issue body or comments)
- [ ] Documentation regarding this issue has been written and merged
- [ ] Details about this issue have been added to the release notes

**Comment by ross-spencer** [comment]
Noticed today as well that if you initiate reingest but  accidentally send the package to the Backlog you end up with a weird workflow. It's possibly a way to mix and match existing and new AIPs? But it's not a particularly documented user story.  Archivematica (by the looks of it) doesn't know it's a reingest once a package emerges from arrangement and you end up with two potentially near-duplicate packages in AIP storage. At least one option to ensure that the workflow for a reingest skips the backlog would be great.

**Comment by sromkey** [comment]
> Noticed today as well that if you initiate reingest but accidentally send the package to the Backlog you end up with a weird workflow. It's possibly a way to mix and match existing and new AIPs? But it's not a particularly documented user story. Archivematica (by the looks of it) doesn't know it's a reingest once a package emerges from arrangement and you end up with two potentially near-duplicate packages in AIP storage. At least one option to ensure that the workflow for a reingest skips the backlog would be great.

Agreed- sending AIPs through reingest to backlog and combining, re-arranging etc. is explicitly not supported (https://www.archivematica.org/en/docs/archivematica-1.12/user-manual/ingest/ingest/#reingest) so it would be good to have a way to prevent this from happening accidentally.
